src/components/Articles.jsx

```diff
@@ -1,9 +1,38 @@
-function Articles(){
-    return (
-        <h3>articles hello</h3>
+import { useEffect,useState } from "react"
+
 
 
-    )
+function Articles(){
+    const [articlesArr, setArticlesArr] = useState([])
+    useEffect(()=>{
+        fetch("https://northcoders-news-api-bjpy.onrender.com/api/articles").then((response)=>{
```

Comment:
use axios here rather than fetch so you don't need to convert the data to json and then have to return it, also, extract this out to its own file and import it into here to be used